tests/functional/confirm.js

```diff
@@ -61,6 +67,48 @@ define([
         // we have to wait until the resent request completes and the
         // success notification is visible
         .then(FunctionalHelpers.testSuccessWasShown(this));
+    },
+
+    'sign up with a gmail address, get the open gmail button': function () {
+      var self = this;
+      var email = 'signin' + Math.random() + '@gmail.com';
+      var PASSWORD = '12345678';
+      var SIGNUP_URL = intern.config.fxaContentRoot + 'signup?context=fx_desktop_v2&service=sync';
+
+      return this.remote
```

Comment:
Can you use `FunctionalHelpers.openPage(SIGNUP_URL, '#fxa-signup-header')` to open the page? It'll reduce the boilerplate, plus allow us to do other customizations like hook up the `listenForFxaCommands` helper universally.